README.md

```diff
@@ -444,6 +444,50 @@ Base = declarative_base()
 To learn more about integrating a database into your FastAPI application,
 follow along the [FastAPI SQL Database guide](https://fastapi.tiangolo.com/tutorial/sql-databases/#create-the-database-models).
 
+#### Django
+
+[Django](https://djangoproject.com)  is a high-level Python web framework that
+encourages rapid development and clean, pragmatic design. 
+
+To use the Cloud SQL Connector with Django, you need to create a custom database
+backend which subclasses the existing MySQL backend. (Note, at present Postgres
+is not supported because Django requires the psycopg2 driver, which is not
+currently compatible with the connector.)
+
+Create a `cloudsql` directory in your project with a blank `__init__.py` and a 
+`base.py` containing the following code:
+
+```python
+from django.db.backends.mysql import base
+from google.cloud.sql.connector import Connector
+
+
+class DatabaseWrapper(base.DatabaseWrapper):
+    def get_new_connection(self, conn_params):
+        return Connector().connect(**conn_params)
```

Comment:
After looking at this a bit closer I think we would want the `Connector()` to be initialized outside the `DatabaseWrapper` class so that it is only called once and can properly cache instances and re-use them.

Something that looks like this...
```python
from django.db.backends.mysql import base
from google.cloud.sql.connector import Connector

connector = Connector()

class DatabaseWrapper(base.DatabaseWrapper):
    def get_new_connection(self, conn_params):
        return connector.connect(**conn_params)
```